### ISSUE TITLE
Python3 support

### ISSUE THREAD

**Issue by rom1sqr** [dmtx/dmtx-wrappers#5]
This project looks dead. But for those wondering how to make a wrapper compatible with Python3,
you just need to replace the `PyMODINIT_FUNC init_pydmtx(void)` function with this :
```
static struct PyModuleDef _pydmtx =
{
    PyModuleDef_HEAD_INIT,
    "_pydmtx",
    "",
    -1,
    dmtxMethods
};

PyMODINIT_FUNC PyInit__pydmtx(void)
{
    return PyModule_Create(&_pydmtx);
}
```
and to remove the `init_pydmtx()` call in the `int main` function. That's it!

**Comment by msva** [comment]
Yeah, project (wrappers) is semi-dead, as all wrappers was made for outdated versions of "backend" languages.

Moreover, for now entire `libdtmx` project is (at least, temporarily) in maintenance-only mode (so, PRs are welcome)
